docs/superpowers/plans/2026-05-30-lambda-scope-graph.md

```
@@ -0,0 +1,1326 @@
+
+**Files:**
+- Create: `lang/lambda/scope/moon.pkg`
+- Create: `lang/lambda/scope/graph.mbt`
+
+- [ ] **Step 1: Create the package config**
+
+Create `lang/lambda/scope/moon.pkg` (this repo uses the `import { ... }` /
+`warnings = "..."` moon.pkg format, NOT `moon.pkg.json` — copy the shape from
+`lang/lambda/edits/moon.pkg`):
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)